pkg/ebpf/c/tracee.bpf.c

```diff
@@ -105,6 +105,7 @@ int KERNEL_VERSION SEC("version") = LINUX_VERSION_CODE;
 #define NET_SEQ_OPS_TYPES   6         // print_net_seq_ops: argument size - TODO: replace with uprobe argument
 #define MAX_KSYM_NAME_SIZE  64
 #define UPROBE_MAGIC_NUMBER 20220829
+#define MEM_DUMP_SIZE       100
```

Comment:
Why 100? Who says that you will be able to read that amount of bytes from a given address?